docs/usage/customization-guide.md

```diff
@@ -620,6 +620,7 @@ The following features are available for matching:
 | **`kernel.config`** | attribute |          |            | Kernel configuration options
 |                  |              | **`<config-flag>`** | string | Value of the kconfig option
 | **`kernel.loadedmodule`** | flag |         |            | Loaded kernel modules
+| **`kernel.enabledmodule`** | flag |        |            | Builtin and dynamic kernel modules
```

Comment:
this documentation does not explain it very well that `enabledmodule` is a superset of `loadedmodule` and what "buildin modules" are

Reply:
Yeah, now that you pointed this out, I agree. Any suggestions how to improve this @mythi? _All enabled kernel modules, i.e. union of loaded dynamic modules and modules built into the kernel_

Reply:
We could also clarify `kernel.loadedmodule` above i.e. with _Loaded dynamic kernel modules_(?)